### PR TITLE
added dlq feature

### PR DESCRIPTION
**Pull request by caferrari**

### CODE REVIEW

**Review by Todev3 — APPROVED** [review]

**Review by juliomoraes — APPROVED** [review]

**Review by henriquegomes6 — APPROVED** [review]

File: src/Queue.ts [hunk 1]
```diff
@@ -18,11 +18,13 @@ export class Queue {
       enableNack: true,
       retryTimeout: 0,
       nackQueue: `${name}.nack`,
```
Comment (tgbaldo) [inline src/Queue.ts:20]: dql ou dlq? hehe

File: src/Queue.ts [hunk 2]
```diff
@@ -18,11 +18,13 @@ export class Queue {
...
+      DQLQueue: `${name}.dlq`,
       retryTopic: `${name}.retry`,
       nackTopic: `${name}.nack`,
       autoDelete: false,
       exclusive: false,
-      prefetch: 1
+      prefetch: 1,
```
Comment (tgbaldo) [inline src/Queue.ts:26]: leater => letter

File: src/Queue.ts [hunk 3]
```diff
@@ -36,6 +38,17 @@ export class Queue {
     return this;
   }
```
Comment (tgbaldo) [inline src/Queue.ts:40]: leater = letter

File: src/interfaces/IQueueOptions.ts [hunk 4]
```diff
@@ -3,11 +3,13 @@ export interface IQueueOptions {
...
   durable: boolean;
   nackQueue: string;
+  DQLQueue: string;
   retryTopic: string;
   nackTopic: string;
   enableNack: boolean;
   retryTimeout: number;
   autoDelete: boolean;
   exclusive: boolean;
   prefetch: number;
```
Comment (tgbaldo) [inline src/interfaces/IQueueOptions.ts:13]: leater = letter

**Pull request merged**